### DISCUSSION TITLE
How can I get student pack as a high school student?

### DISCUSSION THREAD

**Discussion by tofucurd** [community/community#56003] [GitHub Education]
### Hi everyone,

Because I am in high school, so I don't have a education email or an admission letter. What I only have is a handwritten student ID without a seal. Additionally, my school never gave us a transcript.

So, what can I do to get the student pack?

**Comment by samus-aran** [comment]
Hi @tofucurd 

We have seen other members applying from middle or high schools using supplied letters from their principles or head teachers providing evidence they attend on paper using the school's letterhead. I would suggest you speak to your teachers if they would be willing to supply that along with your student ID.

Hope this helps 👍

**Reply by tofucurd** [reply]
thx, I'll try
